server/controllers/user.js

```diff
@@ -39,9 +39,49 @@ function deleteUser(req, res, next) {
 }
 
 /**
+ * Device modifications
+ */
+
+/**
+ * Get device
+ */
+function getDevice(req, res, next) {
+  userService.getDevice(req.user, req.params.deviceId)
+    .then(device => res.json(device))
```

Comment:
Would we rather have smaller amounts of code or readability?
Personally I think that it's worth keeping it as is so that you don't need to check what a method is returning to know what you are returning in the `res.json` call